pkg/kubectl/resource_printer.go

```diff
@@ -2145,22 +2145,75 @@ func printDeploymentList(list *extensions.DeploymentList, w io.Writer, options P
 	return nil
 }
 
+func formatHPAMetrics(specs []autoscaling.MetricSpec, statuses []autoscaling.MetricStatus) string {
+	if len(specs) == 0 {
+		return "<none>"
+	}
+	list := []string{}
+	max := 2
+	more := false
+	count := 0
+	for i, spec := range specs {
+		switch spec.Type {
+		case autoscaling.PodsMetricSourceType:
+			current := "<unknown>"
+			if len(statuses) > i && statuses[i].Pods != nil {
+				current = statuses[i].Pods.CurrentAverageValue.String()
+			}
+			list = append(list, fmt.Sprintf("%s / %s", current, spec.Pods.TargetAverageValue.String()))
+		case autoscaling.ObjectMetricSourceType:
+			current := "<unknown>"
+			if len(statuses) > i && statuses[i].Object != nil {
+				current = statuses[i].Object.CurrentValue.String()
+			}
+			list = append(list, fmt.Sprintf("%s / %s", current, spec.Object.TargetValue.String()))
+		case autoscaling.ResourceMetricSourceType:
+			if spec.Resource.TargetAverageValue != nil {
+				current := "<unknown>"
+				if len(statuses) > i && statuses[i].Resource != nil {
+					current = statuses[i].Resource.CurrentAverageValue.String()
+				}
+				list = append(list, fmt.Sprintf("%s / %s", current, spec.Resource.TargetAverageValue.String()))
```

Comment:
@DirectXMan12 @jszczepkowski This output violates kubectl output conventions.

https://github.com/kubernetes/community/blob/master/contributors/devel/kubectl-conventions.md#output-conventions

The spaces around the `/` make it inconvenient for simple processing.

Reply:
@pwittrock could you please take a look of pr: https://github.com/kubernetes/kubernetes/pull/56331? submitted a fix there.